### PR TITLE
Adding updates to community.md and index.md

### PR DESCRIPTION
**Pull request by mxochicale**
Hi, this PR addresses #31 #24 #32 

Thanks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,9 @@ _site/
 .jekyll-cache/
 .jekyll-metadata
 
+# Pycharm project
+/.idea
```
Comment (paddyroddy) [inline .gitignore:9]: The `/` shouldn't be here, right?
```suggestion
.idea
```

File: index.md [hunk 2]
```diff
@@ -8,14 +8,15 @@ This site documents common tooling we use in Python projects at the [Advanced Re
 
 It is not comprehensive, but intended as a forum to share knowledge across projects.
 Each page contains a table of packages or services that are useful when building a Python package.
-Each entry has a link to the package or service, and a short summary of what it does.
+Each entry has a link to the package or service, and a short summary of what it does, traffic lights and used by.
```
Comment (paddyroddy) [inline index.md:11]: I don't think they all have "used by"

Comment (paddyroddy) [inline index.md:11]: Should move position of "and" for whatever the final sentence is

File: index.md [hunk 3]
```diff
@@ -8,14 +8,15 @@ This site documents common tooling we use in Python projects at the [Advanced Re
...
 
 ## Choosing tools
-If you are working within a larger community, always start with the same tools they recommend - as an example, the [`napari`](https://napari.org/) community have  [`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin) for setting up a new pacakge.
+If you are working within a larger community, always start with the same tools they recommend.
+For example, the [`napari`](https://napari.org/) community have  [`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin) for setting up a new pacakge; as well as [SciKit-Surgery](https://scikit-surgery.github.io/scikit-surgery/) libraries have [Cookiecutter-PythonTemplate](https://github.com/SciKit-Surgery/PythonTemplate) to create new python-based libraries.
```
Comment (paddyroddy) [inline index.md:15]:
```suggestion
For example, the [`napari`](https://napari.org/) community have  [`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin) for setting up a new package; as well as [SciKit-Surgery](https://scikit-surgery.github.io/scikit-surgery/) libraries have [Cookiecutter-PythonTemplate](https://github.com/SciKit-Surgery/PythonTemplate) to create new python-based libraries.
```

Comment (paddyroddy) [inline index.md:15]:
```suggestion
For example, the [`napari`](https://napari.org/) community have the [`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin) for setting up a new pacakge; as well as [SciKit-Surgery](https://scikit-surgery.github.io/scikit-surgery/) libraries have [Cookiecutter-PythonTemplate](https://github.com/SciKit-Surgery/PythonTemplate) to create new python-based libraries.
```

Comment (paddyroddy) [inline index.md:15]:
```suggestion
For example, the [`napari`](https://napari.org/) community have  [`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin) for setting up a new pacakge; and the [SciKit-Surgery](https://scikit-surgery.github.io/scikit-surgery/) library has the [Cookiecutter-PythonTemplate](https://github.com/SciKit-Surgery/PythonTemplate) to create new python-based libraries.
```

**Review by paddyroddy — APPROVED** [review]
Various suggested changes

**Comment by mxochicale** [comment]
Thanks Paddy, just added few changes. Let me know if you have further suggestions.

**Review by paddyroddy — APPROVED** [review]

**Pull request merged**